cc3200tool/cc.py

```diff
@@ -616,9 +619,19 @@ def write_file(self, local_file, cc_filename, sign_file=None, size=0):
             self.erase_file(cc_filename)
 
         alloc_size = max(size, file_len)
+
+        timeout = self.port.timeout
+        if (alloc_size > 200000):
+            timeout = max(timeout, 5 * ((alloc_size / 200000) + 1)) # empirical value is ~252925 bytes for 5 sec timeout
+            if timeout != self.port.timeout:
+                #log.info("File is large, increased timeout %d -> %d", self.port.timeout, timeout)
+				pass
```

Comment:
There are tabs here, but I think the whole `if timeout != self.port.timeout:` branch can be removed